css/preheader.scss

```diff
@@ -0,0 +1,45 @@
+.support {
+    background-color: rgb(47, 28, 106);
+    font-family: sans-serif;
+    padding: 40px 0;
+
+&__h1{
+    font-size: 24px;
+    color: #FFFFFF;
+    line-height: 1.2;
+
+    }
+
+
+&__img{
+    height: 38px;
+    width: 38px;
+    margin-left: 10px;
+     }
+
+
+&__text{
```

Comment:
Ogolnie to nie jest jakas wymagana zasada, ale osobiscie grupuje wlasciwosci CSS wedlug odpowiedzialnosci i zawsze mam szukane wlasciwosci w tym samym miejscu. A grupuje je mniej wiecej tak:

```
.jaks-klasa {
  display: flex;
  align-items: center;
  justifyt-contet: center;

  position: absolute;
  left: 50%;
  top: 50%;

  font-size: 10px;
  font-weight: 700;
  color : red;

  width: 200px;
  height: 200px;
  
  padding: 10px;
  margin ;10px;

  transform: translateX(10xp)'

  opacity: 0.5;
  z-index: 10;

  transition: ...,
}
```